src/hatti/ona/forms.cljs

```diff
@@ -161,19 +161,22 @@
   ([field answer] (format-answer field answer nil))
   ([field answer language] (format-answer field answer language false))
   ([field answer language compact?]
+   (format-answer field answer language false :name))
+  ([field answer language compact? name-or-label]
```

Comment:
I think this function needs to be refactored to take a field and an options map instead of positional arguments, and have a single signature.

Also, `name-or-label` is not a good argument name IMO because it doesn't describe the semantic meaning of the argument. This looks like it's the key fetch from the field, is that correct? We could call it `field-key`.

Propose we have a single signature of:

```clojure
[field answer & {:keys [language compact? field-key] :or {field-key :name}}]
```